src/borg/archiver.py

```diff
@@ -364,8 +364,9 @@ def do_key_export(self, args, repository):
             manager.export_paperkey(args.path)
         else:
             if not args.path:
-                self.print_error("output file to export key to expected")
-                return EXIT_ERROR
+                print("No output path provided. Printing key:")
+                manager.export_paperkey(None)
```

Comment:
maybe it would be even better without that "print", because then the stdout output is exactly the same as borg would write to the on-disk file when a file path has been provided.

Reply:
But there is another big issue:

364 is dealing with the `--paper` part and is happy to work with args.path having any value (including the default, which is None I guess?). So that functionality is already implemented there (with or without a path).

366+ block is the "not paper" branch, so I guess it must not call `export_paperkey` there, but `export_qr` or `export` as seen in 370+. So I guess there 2 have to be chacked whether they can somehow work without a on-disk file as output (but stdout).